### PR TITLE
ExternalWorkerConfig - Fix the json tag for SSHKeySecretReference

### PR DESCRIPTION
**Pull request by ygalblum**

### CODE REVIEW

File: api/v1alpha1/osbuildenvconfig_types.go [hunk 1]
```diff
@@ -97,11 +97,11 @@ type ExternalWorkerConfig struct {
 	// User is the user to connect with to external worker machine
 	// +kubebuilder:validation:Required
 	User string `json:"user"`
-	// RedHatCredsSecretReference is a reference to a secret in the same namespace,
+	// SSHKeySecretReference is a reference to a secret in the same namespace,
```
Comment (bardielle) [inline api/v1alpha1/osbuildenvconfig_types.go:100]: Please update the sample file

Comment (ygalblum) [reply api/v1alpha1/osbuildenvconfig_types.go:100]: The sample file doesn't use this yet. Writing a sample for the External Worker is how I saw this error

Comment (bardielle) [reply api/v1alpha1/osbuildenvconfig_types.go:100]: Also maybe need to edit the README file

Comment (ygalblum) [reply api/v1alpha1/osbuildenvconfig_types.go:100]: Same for the README

Comment (bardielle) [reply api/v1alpha1/osbuildenvconfig_types.go:100]: https://github.com/project-flotta/osbuild-operator/blob/main/config/samples/osbuilder_v1alpha1_osbuildenvconfig.yaml#L16

**Review by bardielle — APPROVED** [review]

**Pull request merged**